packages/search-ui-site-search-connector/src/requestAdapters.js

```diff
@@ -22,21 +32,59 @@ export function adaptFacetConfig(facets) {
   return config;
 }
 
-export function adaptFilterConfig(filterConfig) {
-  if (!filterConfig || !filterConfig.all) return;
+export function adaptFilterConfig(filters) {
+  if (!filters || Object.keys(filters).length === 0) return;
 
-  return filterConfig.all.reduce((acc, filter) => {
-    const [fieldName, fieldValue] = Object.entries(filter)[0];
+  return filters.reduce((acc, filter) => {
+    const fieldName = filter.field;
+    let fieldValue = filter.values;
 
-    if (!acc[fieldName]) {
-      acc[fieldName] = {
-        type: "and",
-        values: []
-      };
+    if (acc[fieldName]) {
+      console.warn(
+        "search-ui-site-search-connector: More than one filter found for a single field"
+      );
+      return acc;
+    }
+
+    if (filter.type && filter.type !== "all") {
+      console.warn(
+        `search-ui-site-search-connector: Unsupported filter type ${
+          filter.type
+        } found, only "all" is currently supported`
+      );
+      return acc;
+    }
+
+    if (fieldValue.find(v => typeof v === "object") !== undefined) {
+      if (fieldValue.length > 1) {
+        console.warn(
+          "search-ui-site-search-connector: Cannot apply more than 1 none-value filters to a single field"
+        );
+        return acc;
+      }
+
+      const firstValue = fieldValue[0];
+      if (
+        firstValue.from ||
```

Comment:
An alternative to this could be `if (Number.isInteger(firstValue.from) || Number.isInteger(.to) {` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isInteger